meinberlin/apps/budgeting/assets/__tests__/ControlBarDropdown.jest.jsx

```diff
@@ -52,3 +52,37 @@ test('ControlBarDropdown is positioned right', () => {
   const allChoiceElement = screen.getByText('all')
   expect(allChoiceElement).toBeTruthy()
 })
+
+test('ControlBarDropdown without current selection', () => {
+  const onSelectFilterFn = jest.fn()
+  render(
+    <ControlBarDropdown
+      filter={{
+        position: 'right',
+        label: 'categories',
+        icons: [['1', 'some/url']],
+        choices: [['1', 'all'], ['2', 'category1']]
+      }}
+      onSelectFilter={onSelectFilterFn}
+    />
+  )
+  expect(onSelectFilterFn).toHaveBeenCalled()
+})
+
+test('ControlBarDropdown with default selection', () => {
+  const onSelectFilterFn = jest.fn()
+  render(
+    <ControlBarDropdown
+      filter={{
+        position: 'right',
+        default: 'all',
+        label: 'categories',
+        icons: [['1', 'some/url']],
+        choices: [['1', 'all'], ['2', 'category1']]
+      }}
+      onSelectFilter={onSelectFilterFn}
+    />
+  )
+  screen.debug()
```

Comment:
what is this for / what does it do?

Reply:
oh, this prints the DOM that is created by jest.
Just used to debug while writing the test. Should be removed. Will remove it..